packages/knip/src/plugins/biome/index.ts

```diff
@@ -0,0 +1,49 @@
+import type { IsPluginEnabled, Plugin, PluginOptions, ResolveConfig } from '../../types/config.js';
+import { toDeferResolve, toDependency, toIgnore, type ConfigInput, type Input } from '../../util/input.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { BiomeConfig } from './types.js';
+
+const title = 'biome';
+
+const enablers = ['@biomejs/biome'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => 
+  {
+    const enabled = hasDependency(dependencies, enablers);
+    return enabled;
+  }
```

Comment:
Formatting feels a bit off, please make sure to run `bun run format`

Reply:
Yeah I must have misread it somewhere and ran `npm run format` from the root folder instead. I just ran `bun run format` now.